package.json

```diff
@@ -7,21 +7,20 @@
   "repository": "https://github.com/probot/stale",
   "scripts": {
     "start": "probot run ./index.js",
-    "test": "mocha && xo --extend probot"
+    "test": "standard"
```

Comment:
@bkeepers was there any reasoning for adding mocha in #56? When I ran npm test with `mocha && standard` I got errors because mocha can't find any test files.

Reply:
Oops, that was an accident. I was working on a branch to add tests (😲) and didn't mean to commit that yet.

Reply:
😮 tEsTs?? 😱